plugins/modules/dcnm_policy.py

```diff
@@ -1070,9 +1104,9 @@ def dcnm_policy_send_message_to_dcnm(self):
                 and (resp.get("DATA", None) is not None)
             ):
                 if resp["DATA"].get("successList", None) is not None:
-                    if "is created successfully" in resp["DATA"]["successList"][0].get(
-                        "message"
-                    ):
+                    if "is created successfully" in resp["DATA"][
```

Comment:
Would it be better to check the return code here instead of the message?  Just worried that the string message could change in the future and this check would fail

Reply:
this is the only string available and it is same between versions. Otherwise we may have a RETURN CODE 200, but there may be a case of failure. Added this specifically to check for that. Also this was already there in the released code.